bin/cml-runner.js

```diff
@@ -256,11 +267,29 @@ const run = async (opts) => {
     await tf.save_tfstate({ tfstate, path });
   }
 
+  // if (name !== NAME) {
+  //   console.log(
+  //     'The --name option is deprecated: please use --labels to differentiate runners instead.'
+  //   );
+  // }
+
   await cml.repo_token_check();
-  if (await cml.runner_by_name({ name }))
-    throw new Error(
-      `Runner name ${name} is already in use. Please change the name or terminate the other runner.`
-    );
+
+  if (reuse_existing && (await cml.runners_by_labels({ labels }))) {
```

Comment:
true && [] && console.log('here!')

Needs the length check

Reply:
🙏  Thanks! `[] && console.log("🤦 ")` Sorry for the lapsus, my brain was damaged by snake taming. 🐍

Reply:
Today I learned that `([] == false) === true` and `!![] == true` despite `Boolean([]) == true` 🤯

Reply:
JS the good parts by Douglas Crockford. Highly recommended